packages/frontend/src/pages/Invite.tsx

```diff
@@ -187,10 +186,14 @@ const Signup: FC = () => {
             <Helmet>
                 <title>Register - Lightdash</title>
             </Helmet>
-            <FormWrapper>
-                <LogoWrapper>
-                    <Logo src={LightdashLogo} alt="lightdash logo" />
-                </LogoWrapper>
+            <Stack w={400} mt="xl" pt="lg">
```

Comment:
can you mark places like this with FIXME? it will be hard to trace those once we add proper sizes

Reply:
done!

Reply:
tks! 🙌